packages/plugins/graphql/server/src/services/internals/types/publication-status.ts

```diff
@@ -0,0 +1,23 @@
+import { enumType } from 'nexus';
+import type { Context } from '../../types';
+
+export default ({ strapi }: Context) => {
+  const { PUBLICATION_STATUS_TYPE_NAME } = strapi.plugin('graphql').service('constants');
+
+  return {
+    /**
+     * An enum type definition representing a publication state
+     * @type {NexusEnumTypeDef}
+     */
+    PublicationStatus: enumType({
+      name: PUBLICATION_STATUS_TYPE_NAME,
+
+      members: {
+        // Published only
```

Comment:
Draft only

Reply:
bad copy paste the name are clear enough I'll remove the comments